backend/pick-git/src/main/java/com/woowacourse/pickgit/user/domain/follow/Followers.java

```diff
@@ -16,13 +15,13 @@
         cascade = CascadeType.PERSIST,
         orphanRemoval = true
     )
-    private List<Follow> followers = new ArrayList<>();
+    private List<Follow> followers;
 
-    public Followers() {
+    protected Followers() {
     }
```

Comment:
```suggestion
    public Followers() {
        this(new ArrayList<>());
    }
```

난 개인적으로 편의를 위해 이렇게 열어놨어 ㅋㅋ